### PR TITLE
Support shell command activity

### PR DESCRIPTION
**Pull request by anniezhang77**
risk: low

### CODE REVIEW

File: spade-core/src/main/scala/com/salesforce/mce/spade/SpadeContext.scala [hunk 1]
```diff
@@ -33,6 +34,7 @@ object SpadeContext {
       config.getInt("max-attempt"),
       config.getInt("delivery-timeout"),
       config.getInt("execution-timeout"),
+      config.getString("log-uri"),
```
Comment (realstraw) [inline spade-core/src/main/scala/com/salesforce/mce/spade/SpadeContext.scala]: is this required or optional?

Comment (anniezhang77) [reply spade-core/src/main/scala/com/salesforce/mce/spade/SpadeContext.scala]: it's required

Comment (realstraw) [reply spade-core/src/main/scala/com/salesforce/mce/spade/SpadeContext.scala]: can we make it optional? Orchard should have a default configuration there if missing.

Comment (anniezhang77) [reply spade-core/src/main/scala/com/salesforce/mce/spade/SpadeContext.scala]: this is not for orchard logs, but job logs on s3. do we still wanna have a default in orchard?

Comment (anniezhang77) [reply spade-core/src/main/scala/com/salesforce/mce/spade/SpadeContext.scala]: correct me if i'm wrong: data pipeline (orchard) doesn't know where job logs should be stored, it just takes command and execute it. hyperion (spade) decides where to put the logs as default. is that right?

Comment (realstraw) [reply spade-core/src/main/scala/com/salesforce/mce/spade/SpadeContext.scala]: https://github.com/salesforce/orchard/blob/master/orchard-provider-aws/src/main/resources/reference.conf#L3 this is the job log URI that can be set as default (slightly different from AWS data pipeline), spade should be able to override it if it decides to do so.

Comment (anniezhang77) [reply spade-core/src/main/scala/com/salesforce/mce/spade/SpadeContext.scala]: then we'll need to make it required in orchard, it's optional at the moment.

Comment (realstraw) [reply spade-core/src/main/scala/com/salesforce/mce/spade/SpadeContext.scala]: i mean it doesn't have to fail if we don't specify it, is it required by the SSM or we just made it in orchard where we could have made it optional?

Comment (anniezhang77) [reply spade-core/src/main/scala/com/salesforce/mce/spade/SpadeContext.scala]: it's required by ssm.

Comment (anniezhang77) [reply spade-core/src/main/scala/com/salesforce/mce/spade/SpadeContext.scala]: updated to optional

**Review by damianxu88 — APPROVED** [review]
LGTM

File: version.sbt [hunk 2]
```diff
@@ -1 +1 @@
- ThisBuild / version := "0.6.1"
+ ThisBuild / version := "0.6.2"
```
Comment (realstraw) [inline version.sbt]: let's change this to 0.7.0

**Review by schowsf — APPROVED** [review]
LGTM

File: version.sbt (hunk 2 above)
Comment (anniezhang77) [reply version.sbt]: done

**Review by realstraw — APPROVED** [review]

**Pull request merged**